src/content/docs/tips/shadcn.mdx

```diff
@@ -0,0 +1,266 @@
+        ```sh
+        bunx astro add tailwind
+        ```
+      </TabItem>
+    </Tabs>
+3. Create a `styles/globals.css` file in your `src/pages/index.astro` file
+
+        ```css title="src/styles/globals.css"
+        @tailwind base;
+        @tailwind components;
+        @tailwind utilities;
+        ```
+4. Import the `styles/globals.css` file in the `src/pages/index.astro` file:
+
+        ```astro title="src/pages/index.astro"
+        ---
+        import '@/styles/globals.css'
+        ---
+        ```
+5. Update `astro.config.mjs` and set `applyBaseStyles` to `false`:
+
+        ```js title="astro.config.mjs"
+            export default defineConfig({
+            integrations: [
+                tailwind({
+                applyBaseStyles: false,
+                }),
+            ],
+        });
+        ```
+6. Edit tsconfig.json file
+    Add the following code to the `tsconfig.json` file to resolve paths:
+    
+        ```ts title="tsconfig.json" {4-9} showLineNumbers
+        {
+            "compilerOptions": {
+                // ...
+                "baseUrl": ".",
+                "paths": {
+                "@/*": [
+                    "./src/*"
+                ]
+                }
+            }
+        }
+    ```
+
+7. Run the CLI
+    <Tabs syncKey="shadcn">
+      <TabItem label="npm" icon="seti:npm">
+        ```sh
+       npx shadcn@latest init
+        ```
+      </TabItem>
+      <TabItem label="pnpm" icon="pnpm">
+        ```sh
+        pnpx shadcn@latest init
+        ```
+      </TabItem>
+      <TabItem label="yarn" icon="seti:yarn">
+        ```sh
+        npx shadcn@latest init
+        ```
+      </TabItem>
+      <TabItem label="bun" icon="bun">
+        ```sh
+        bunx shadcn@latest init
+        ```
+      </TabItem>
+    </Tabs>
+10. That's it, you're all set up to use Shadcn UI in your project!
+    You can now start adding components from the Shadcn UI library to your project.
+    
+        <Tabs syncKey="shadcn">
+      <TabItem label="npm" icon="seti:npm">
+        ```sh
+       npx shadcn@latest add button
+        ```
+      </TabItem>
+      <TabItem label="pnpm" icon="pnpm">
+        ```sh
+        pnpx shadcn@latest add button
+        ```
+      </TabItem>
+      <TabItem label="yarn" icon="seti:yarn">
+        ```sh
+        npx shadcn@latest add button
+        ```
+      </TabItem>
+      <TabItem label="bun" icon="bun">
+        ```sh
+        bunx shadcn@latest add button
+        ```
+      </TabItem>
+    </Tabs>
+
+        The command above will add the `Button` component to your project. You can then import it like this:
+
+        ```astro title="src/pages/index.astro" {2,10}
+        ---
+        import { Button } from "@/components/ui/button"
+        ---
+
+        <html lang="en">
+        <head>
+            <title>Astro</title>
+        </head>
+        <body>
+            <Button>Hello World</Button>
+        </body>
+        </html>
+        ```
+</Steps>
```

Comment:
This section essentially duplicates the [shadcn astro docs](https://ui.shadcn.com/docs/installation/astro), I think in order to keep this tip on topic we can just say follow that getting started

Reply:
Yes I agree with you. I think this tip should
1. point to an example
2. point to the official docs
3. explain tht common mistakes, eg. react context not being shared between islands hence the need to move stuff to its dedicated island